presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -504,7 +504,7 @@ private MaterializedResult executeInternal(Session session, @Language("SQL") Str
             TaskContext taskContext = createTaskContext(executor, session);
 
             List<Driver> drivers = createDrivers(session, sql, outputFactory, taskContext);
-            drivers.stream().map(closer::register);
+            drivers.forEach(closer::register);
```

Comment:
Consider removing "fixup!" from the cmt msg.

Also, would it possible to catch `drivers.stream().map(closer::register)` with a static analysis? Stream is Autoclosable and left not closed.